sigs/cncf-sigs.md

```diff
@@ -0,0 +1,252 @@
+
+### SIG Member Roles
+
+#### Chair
+
+* Three chairs where the active chair rotates each week/fortnight/month.
+
+* Primarily performs administrative functions including collecting and compiling topics for the (bi)weekly agenda, chairing the meeting, ensuring that quality meeting minutes are published, and follow-up actions tracked and resolved.
+
+* A chair role may be held and performed by a tech lead, in cases where one person has the time and ability to perform both roles to the satisfaction of the TOC and SIG members.
+
+#### Tech Lead
+
+* Leads projects in the SIG’s area.
+
+* Has the time and ability to perform deep technical dives on projects.   Projects may include formal CNCF projects or other projects in the area covered by the SIG.
+
+#### Other named roles
+
+* Named and defined by the SIG (e.g. scribe, PR lead, docs/training lead, etc)
+
+* Approved by supermajority of the chairs.
+
+#### Other members
+
+* Self-declared
+
+* May either have no explicit roles or responsibilities, or formally assigned roles (see above).
+
+* May not create the impression that they have any authority or formal responsibilities in the SIG other than assigned roles.
+
+### Elections
+
+* The TOC nominates Chairs
+
+* Chairs are assigned following a 2/3 majority vote of the TOC
+
+* Terms last for 1 year but staggered such that at least 1 of the chairs is able to maintain continuity
+
+* The TOC and Chairs nominate Tech leads
+
+* Tech leads are assigned following a majority vote of the TOC and SIG Chairs
+
+* SIG Chairs and Tech Leads may be unassigned from the SIG at any time following a 2/3 majority vote of the TOC
+
+### Governance
+
+* All SIGs inherit and follow the CNCF TOC Operating Principles.
+
+* SIGs must have a documented governance process that encourages community participation and clear guidelines to avoid biased decision-making.  
+
+    * NOTE: aim here is to align with "minimal viable" model of the CNCF projects, and only have such governance as is needed, not anything too burdensome
+
+* They may grow a set of practices over time in the same way as an OSS Project, provided this is consistent with CNCF Operating Principles. 
+
+* As with CNCF Projects all exceptions and disputes are handled by TOC with CNCF Staff help
+
+### Budget & Resource
+
+* No formal systematic budget at this time, other than commitment of CNCF executive staff to provide named person as liaison point.
+
+* Just as CNCF Projects may have "help" offered by CNCF and may ask for things via the [ServiceDesk](https://github.com/cncf/servicedesk), the SIGs may do this.
+
+## Retirement
+
+* In the event that a SIG is unable to regularly establish quorum, or fulfill the responsibilities and/or regularly report to the TOC, the TOC will:
```

Comment:
We don't define quorum in this doc, nor do we define what it is used for? Meeting attendance? Do SIG meetings have to have a certain # of people attend before they can approve an action?  We should be clear on the intent here.

Reply:
Good point.  I think we should cover that in the detailed governance guidelines for the SIG's, rather than in this document.